src/components/admin/order/realtime/modal/order-by-product/OrderByProductModal.tsx

```diff
@@ -95,33 +127,42 @@ function OrderByProductModal({ orders, isModalOpen }: OrderByProductModalProps)
     .sort((a, b) => b.count - a.count);
 
   return (
-    <ModalContainer>
-      <HeaderContainer>
-        <AppLabel size={20} style={{ fontWeight: 700 }}>
-          실시간 상품별 주문량
-        </AppLabel>
-        <AppLabel size={14}>*결제 완료 주문만 표시됩니다.</AppLabel>
-      </HeaderContainer>
-      <ProductContainer>
-        {sortedProducts.map(({ productId, count }) => {
-          const product = productMap[productId];
-
-          if (!product || count <= 0) return null;
-
-          return (
-            <ProductItem key={productId}>
-              <ProductImage src={product.imageUrl} alt={product.name} />
-              <ProductInfoContainer>
-                <AppLabel size={14} style={{ fontWeight: 700 }}>
-                  {product.name}
-                </AppLabel>
-                <AppLabel size={14}>{count}개</AppLabel>
-              </ProductInfoContainer>
-            </ProductItem>
-          );
-        })}
-      </ProductContainer>
-    </ModalContainer>
+    <ProductModalContainer>
+      <Overlay isOpen={isModalOpen} onClick={closeModal} />
+      <SidebarContainer isOpen={isModalOpen}>
+        <SidebarHeader>
+          <CloseButton onClick={closeModal}>
+            <RiCloseLine size={24} />
+          </CloseButton>
+
+          <TitleContainer>
+            <AppLabel size={18} style={{ fontWeight: 700 }}>
+              실시간 상품별 주문량
+            </AppLabel>
+            <AppLabel size={16}>*결제 완료 주문만 표시됩니다.</AppLabel>
+          </TitleContainer>
+        </SidebarHeader>
+        <ProductContainer>
+          {sortedProducts.map(({ productId, count }) => {
+            const product = productMap[productId];
+            const productImageUrl = product.imageUrl;
+            const productName = product.name;
+
+            if (!product || count <= 0) return null;
```

Comment:
```suggestion
            if (count <= 0) return null;
```

상품은 없더라도 보여줘야 합니다! (삭제되었어도 주문이 들어왔기 때문입니다~)